### ISSUE TITLE
qr code not showing on launch

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#3238]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
try {
	execSync(cmd, {stdio: 'inherit'})
}  catch (error) {
    console.log(`Status Code: ${error.status} with '${error.message}'`);
 }
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN && npm run types:publish `);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.72.1](https://github.com/open-wa/wa-automate-nodejs/compare/4.72.0...4.72.1) (2024-08-21)

- Release 4.72.1 [`453b458`](https://github.com/open-wa/wa-automate-nodejs/commit/453b458c1c236eade567a610d88ed1ede13c3fde)
- 🔊 : added better error logging to chatwoot integration [`20b8de0`](https://github.com/open-wa/wa-automate-nodejs/commit/20b8de0045ca2f68e7b2b56f784c03fde84a6060)
- 🚑 fix qr issue #3238 [`23d1d01`](https://github.com/open-wa/wa-automate-nodejs/commit/23d1d014a219f17de2d2d5b4b3cbcf4aad066808)
- 📝  : updated link code example in documentation to reflect correct formatting and value [`c59bf23`](https://github.com/open-wa/wa-automate-nodejs/commit/c59bf238f2ab4af1280b6a5aa61ac130dcd90591)
- updated types-only package [`a6d64e8`](https://github.com/open-wa/wa-automate-nodejs/commit/a6d64e854c8ca3345d6bcc65a43c48dddefecfc6)
